ark/type/type.ts

```diff
@@ -96,7 +101,18 @@ export interface TypeParser<$ = {}> extends Ark.boundTypeAttachments<$> {
 	define: DefinitionParser<$>
 	generic: GenericParser<$>
 	schema: SchemaParser<$>
+	/**
+	 * Construct a literal (or by-reference object) type
```

Comment:
Create a Type that is satisfied only by a value `===` to the argument passed to this function.